common/planners/src/plan_node_display_indent.rs

```diff
@@ -304,4 +306,9 @@ impl<'a> PlanNodeIndentFormatDisplay<'a> {
     fn format_copy(f: &mut Formatter, plan: &CopyPlan) -> fmt::Result {
         write!(f, "{:?}", plan)
     }
+
+    fn format_call(f: &mut Formatter, _plan: &CallPlan) -> fmt::Result {
+        // TODO
+        write!(f, "Call")
```

Comment:
Maybe we can print the args,like `CALL <name>(arg1, arg2, ...)`

Reply:
SGTM